cluster/gce/upgrade-aliases.sh

```diff
@@ -35,8 +35,9 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 # Assumed vars:
 #   PROJECT
 function get-k8s-node-routes-count() {
-  local k8s_node_routes_count=$(gcloud compute routes list \
-    --project=${PROJECT} --filter='description=k8s-node-route' \
+  local k8s_node_routes_count
+  k8s_node_routes_count=$(gcloud compute routes list \
+    --project="${PROJECT}" --filter='description=k8s-node-route' \
```

Comment:
change reason:
```
In cluster/gce/upgrade-aliases.sh line 38:
  local k8s_node_routes_count=$(gcloud compute routes list \
        ^-- SC2155: Declare and assign separately to avoid masking return values.
```